review/tests.py

```diff
@@ -8,12 +8,26 @@
 DOG_OWNER_ID = 1
 
 
-@pytest.mark.django_db
 @pytest.fixture
 def test_review():
     return Review.create(review=REVIEW, rating=RATING, daycare_id=DAY_CARE_ID, dogowner_id=DOG_OWNER_ID)
 
 
+@pytest.fixture
+def test_daycare():
+    return DayCare.create(email="daycare_review@gmail.com", username="daycare_review", password="pass",
+                          name="reviewDayCare", description="This is test daycare for review", price_per_day=10,
+                          capacity=50, area="Merkaz", city="Tel-Aviv", address="The best review")
+
+
+@pytest.fixture
+def test_dogowner():
+    return DogOwner.create(email="dogowner_review@gmail.com", username='dogowner_review', password='pass',
```

Comment:
we are creating those in most models. maybe we can make use of conftest?  (hint: it can work on different levels)

Reply:
I created conftest for the review model and created #80 issue to create one conftest for all models